brush-shell/tests/cases/builtins/builtin.yaml

```diff
@@ -15,8 +15,8 @@ cases:
 
   - name: "builtin passing through results"
     stdin: |
-      builtin false; echo "builtin false => $?"
-      builtin true; echo "builtin true => $?"
+      builtin false --help; echo "builtin false => $?"
```

Comment:
Thanks for adding to the tests.

This test case happens to call `false`/`true`, but it's really focused on testing `builtin`. Will you please:

* Revert the change to this file to keep this test simple.
* Add a new file in this directory called something like `true_false.yaml` that defines a new test set
* In that *new* file add test cases that cover the usage you're fixing with this PR (e.g., basic `false`/`true` usage, usage with non-dash arguments, usage with `--help` or other dash-arguments)

Reply:
`false --help` is fine.